src/routes/registration.ts

```diff
@@ -45,8 +51,27 @@ export const createRegistrationRoute: RouteCreator =
       .then(({ data: flow }) => {
         // Render the data using a view (e.g. Jade Template):
         res.render("registration", {
-          ...flow,
-          signInUrl: initLoginUrl,
+          nodes: flow.ui.nodes,
+          webAuthnHandler: filterNodesByGroups({
+            nodes: flow.ui.nodes,
+            groups: ["webauthn"],
+            attributes: ["button"],
+            withoutDefaultAttributes: true,
```

Comment:
Not really related to this PR, but negative flags/params/attributes always seem a bit "smelly" to me. Instead, I'd prefer `withDefaultAttributes` and handle its omission accordingly (e.g. determine a good default value/behavior for it). WDYT? Can we still change that?

Reply:
Yeah I understand what you mean. We cannot change this now since this is in a library which had to get tested for this. I think in future we can update it, but for now the priority is to get this released.
https://github.com/ory/integrations/blob/main/src/ui/index.ts#L142-L149

Reply:
Oh, I missed that this is part of an older library. Alright. 

Thought this is part of `elements`...

Reply:
all good :)